.github/actions/setup-job/action.yml

```diff
@@ -70,11 +72,19 @@ runs:
       if: inputs.deps
       run: |
         if [[ "${{ inputs.dart }}" == "true" || "${{ inputs.flutter }}" == "true" ]] && [[ "${{ inputs.rust }}" == "true" ]]; then
-          # ffigen dep, not handled by `just deps` as it's OS specific.
+          # ffigen dep, not handled by `just` as it's OS specific.
           sudo apt-get install libclang-10-dev
-          just deps
```

Comment:
`just deps` is more then just dart and flutter deps, but also `install-async-bindgen`

Reply:
I saw it's handled different in the `justfile` now, through then `sudo apt-get install libclang-10-dev` probably should move into the `[ "${{ inputs.dart }}" == "true" ]` branch.

Also when flutter was added in a previous commit, it messed up the rust checks it should be `"${{ inputs.rust }}" != "false"` as rust can be `true`,`false` or `nightly`. It just happens to not cause problems currently as we only use `nightly` with `deps == false`.